lib/utilities/reportHandler.js

```diff
@@ -96,38 +133,53 @@ function getCRRStats(log, cb) {
     const routes = backbeat.routes(redisKeys, sites);
     const details = routes.find(route =>
         route.category === 'metrics' && route.type === 'all');
-    // Add `site` as we're not using Backbeat's request parser for the API's URI
-    details.site = 'all';
-    return backbeatMetrics.getAllMetrics(details, (err, res) => {
+    return async.parallel({
+        total: done => _crrRequest(backbeatMetrics, details, 'all', log, done),
+        byLocation: done => async.map(sites,
+            (site, next) => _crrRequest(backbeatMetrics, details, site, log,
+                (err, res) => {
+                    if (err) {
+                        log.debug('err should be null', {
```

Comment:
message should be more explicit (like failed to get site-specific crr stats) and include the site in the extra properties